.travis.yml

```diff
@@ -101,12 +101,16 @@ script:
   - |
     if [[ -v HIVE_TESTS ]]; then
       presto-hive-hadoop2/bin/run_hive_tests.sh
-      if [[ -v S3_TESTS_AWS_ACCESS_KEY_ID ]]; then
-        env AWS_ACCESS_KEY_ID=$S3_TESTS_AWS_ACCESS_KEY_ID \
-        AWS_SECRET_ACCESS_KEY=$S3_TESTS_AWS_SECRET_ACCESS_KEY \
+      if [[ -v HIVE_TESTS_AWS_ACCESS_KEY_ID ]]; then
+        env AWS_ACCESS_KEY_ID=$HIVE_TESTS_AWS_ACCESS_KEY_ID \
+        AWS_SECRET_ACCESS_KEY=$HIVE_TESTS_AWS_SECRET_ACCESS_KEY \
         S3_BUCKET_ENDPOINT=$S3_TESTS_BUCKET_ENDPOINT \
         S3_BUCKET=$S3_TESTS_BUCKET \
         presto-hive-hadoop2/bin/run_hive_s3_tests.sh
```

Comment:
If you want to reuse this `if` then add another nested if here that `HIVE_TESTS_AWS_ACCESS_KEY_ID` and `HIVE_TESTS_AWS_SECRET_ACCESS_KEY` are set before launching s3 tests

Reply:
I was intending on reusing the same AWS key for both S3 & Glue access which is why I renamed ```S3_TESTS_AWS_ACCESS_KEY_ID``` to something more generic like ```HIVE_TESTS_AWS_ACCESS_KEY_ID```.  

I wasn't sure how or where this key is set within Travis but anyways, using ```S3_TESTS_AWS_ACCESS_KEY_ID``` can work as well for accessing Glue as long as the AWS user for that key has permission to Glue APIs.